plugins-extra/query_enhancements/public/query_assist/components/submit_button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { EuiButtonIcon } from '@elastic/eui';
+
+interface SubmitButtonProps {
+  isDisabled: boolean;
+}
+
+export const QueryAssistSubmitButton: React.FC<SubmitButtonProps> = (props) => {
+  return (
+    <EuiButtonIcon
+      iconType="returnKey"
+      display="base"
+      isDisabled={props.isDisabled}
+      size="s"
+      type="submit"
+      aria-label="submit-question"
```

Comment:
do we want to make this more detailed? imo i think it's fine as is for [accessibility](https://oui.opensearch.org/1.6/#/guidelines/accessibility) purposes but i'm not expert. perhaps we can get u/x feedback on what the aria label should be. like should be "Submit question to query assistant".